app/com/linkedin/drelephant/exceptions/util/ExceptionUtils.java

```diff
@@ -103,12 +104,17 @@ public static String getJobName(String jobExecUrl) {
     return null;
   }
 
-    public static void debugLog(String message) {
+  public static void debugLog(String message) {
     if (debugEnabled) {
       logger.debug(message);
     }
   }
 
+  public static int getLogSimilarityPercentage(String log1, String log2) {
```

Comment:
can u please write test case for just this method . I know , this has been tested through some methods . But its better to have both positive and negative test cases for this method (since this is the heart of the logic).  Specifically i want to see , logs with same exception but having different container ID , will they be removed.

Reply:
For verifying if similar logs(only difference is the container_id) removal you can refer [log_1](https://github.com/linkedin/dr-elephant/pull/660/files#r409060096) and [log_2](https://github.com/linkedin/dr-elephant/pull/660/files#r409060076) which are analyzed in [Test](https://github.com/linkedin/dr-elephant/pull/660/files#r409063339).

Reply:
ok got it . I meant to have unit test case for just this method , so that I can see what are the cases tested . Also can u put some negative test cases ,where they are not simillar and it result less similarity percentage